clippy_lints/src/derivable_impls.rs

```diff
@@ -85,6 +85,17 @@ fn contains_trait_object(ty: Ty<'_>) -> bool {
     }
 }
 
+fn determine_derive_macro(cx: &LateContext<'_>, is_const: bool) -> Option<&'static str> {
+    if is_const {
+        if !cx.tcx.features().enabled(sym::derive_const) {
+            return None;
+        }
+        return Some("derive_const");
+    }
+    Some("derive")
+}
```

Comment:
This can be made more concise:

```suggestion
    if is_const {
        cx.tcx.features().enabled(sym::derive_const).then_some("derive_const")
    } else {
        Some("derive")
    }
```

Reply:
Thank you!